cli/cliui/table_test.go

```diff
@@ -52,7 +52,7 @@ type tableTest3 struct {
 func Test_DisplayTable(t *testing.T) {
 	t.Parallel()
 
-	someTime := time.Date(2022, 8, 2, 15, 49, 10, 0, time.Local)
+	someTime := time.Date(2022, 8, 2, 15, 49, 10, 0, time.UTC)
```

Comment:
Does it cause flakiness or fail permanently under specific conditions? I guess that you can add a warning/comment here to discourage using `time.Local`.

Reply:
It's permanent. I don't feel a comment necessary here since this was a test-bug. The test expects UTC output but is relying on system local time.